cmd/root.go

```diff
@@ -134,7 +134,7 @@ func Execute() error {
 		return nil
 	})
 
-	csdaemon.NotifySystemd(log.StandardLogger())
+	csdaemon.Notify("READY=1", log.StandardLogger())
```

Comment:
might need @mmetc to weigh in on this change

Reply:
it's ok

Reply:
actually crowdsec does 

csdaemon.Notify(csdaemon.Ready, log.StandardLogger())